modeling-cmds/src/datetime.rs

```diff
@@ -0,0 +1,42 @@
+use kittycad_execution_plan_traits::{MemoryError, NumericPrimitive, Primitive, Value};
+
+/// A wrapper for chrono types, since we need to impl Value for them.
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct DateTimeLocal {
+    value: chrono::DateTime<chrono::Local>,
+}
+
+impl Value for DateTimeLocal {
+    fn into_parts(self) -> Vec<Primitive> {
+        vec![Primitive::NumericValue(NumericPrimitive::Integer(
+            self.value.timestamp(),
+        ))]
+    }
+
+    /// Read the value from memory.
+    fn from_parts<I>(values: &mut I) -> Result<(Self, usize), MemoryError>
+    where
+        I: Iterator<Item = Option<Primitive>>,
+    {
+        let maybe_datetime = values.next().unwrap();
+
+        match maybe_datetime {
+            None => Ok((
```

Comment:
Seems like your `into_parts()` method will never write `None` so I think this should return MemoryError if it finds a `None`.